src/main/webapp/package.json

```diff
@@ -0,0 +1,47 @@
+{
```

Comment:
Pare this down to what you need, e.g. you don't have end to end tests yet so no need for "e2e", and you're not using most of those dependencies so remove them.